config/package_resouce.go

```diff
@@ -14,26 +14,229 @@
 
 package config
 
-import agentendpointpb "github.com/GoogleCloudPlatform/osconfig/internal/google.golang.org/genproto/googleapis/cloud/osconfig/agentendpoint/v1alpha1"
+import (
+	"errors"
+	"fmt"
+
+	agentendpointpb "github.com/GoogleCloudPlatform/osconfig/internal/google.golang.org/genproto/googleapis/cloud/osconfig/agentendpoint/v1alpha1"
+	"github.com/GoogleCloudPlatform/osconfig/packages"
+)
 
 type packageResouce struct {
 	*agentendpointpb.ApplyConfigTask_Config_Resource_PackageResource
+
+	policy ManagedPackage
+}
+
+// AptPackage describes an apt package policy.
```

Comment:
The word policy might lead to some confusion since we already have an os policy

Reply:
This is just the comment, and it is because of the os policy, just the package resource in that policy. The reason these are public is so we can run conflict resolution later on

Reply:
Got it.
Should the comment be reworded as "AptPackage describes an apt package **_resource_**"
Same for all other comments for other package types.

Reply:
done